### PR TITLE
docs: Added release notes for #5262

### PR DESCRIPTION
**Pull request by ogabrielides**
## What was done?
Added release notes for #5262.

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

File: doc/release-notes-5262.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Version 3 of Coinbase will include the following two fields:
+- bestCLHeightDiff (uint32)
+- bestCLSignature (BLSSignature)
+
+bestCLSignature will hold the best Chainlock signature known at the moment of mining.
+bestCLHeightDiff is equal to the diff in heights from the mined block height to the actual Chainlocked block height.
+
+Although miners are forced to include version 3 Coinbase, actual real-time best Chainlock signature isn't required: as long as blocks will contain at least equal (or newer) bestCLSignature than the previous block.
+
+Note: Until the first bestCLSignature included in Coinbase once v20 activates, null bestCLSignature and bestCLHeightDiff are perfectly valid.
```
Comment (PastaPastaPasta) [inline doc/release-notes-5262.md:15]:
```suggestion
Note: Until the first bestCLSignature included in Coinbase once v20 activates, null bestCLSignature and bestCLHeightDiff are perfectly valid.

```

File: doc/release-notes-5262.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Once v20 activates, miners will produce blocks containing a newer version (=3) of Coinbase transaction.
+
+Version 3 of Coinbase will include the following two fields:
+- bestCLHeightDiff (uint32)
+- bestCLSignature (BLSSignature)
+
+bestCLSignature will hold the best Chainlock signature known at the moment of mining.
+bestCLHeightDiff is equal to the diff in heights from the mined block height to the actual Chainlocked block height.
+
+Although miners are forced to include version 3 Coinbase, actual real-time best Chainlock signature isn't required: as long as blocks will contain at least equal (or newer) bestCLSignature than the previous block.
```
Comment (thephez) [inline doc/release-notes-5262.md:13]:
```suggestion
Although miners are forced to include a version 3 Coinbase, the actual real-time best Chainlock signature isn't required as long as blocks contain a bestCLSignature equal to (or newer than) the previous block.
```

File: doc/release-notes-5262.md (hunk 1 above)
Comment (thephez) [inline doc/release-notes-5262.md:15]:
```suggestion
Note: Until the first bestCLSignature is included in a Coinbase once v20 activates, null bestCLSignature and bestCLHeightDiff values are perfectly valid.
```

**Review by thephez — CHANGES_REQUESTED** [review]

**Review by thephez — APPROVED** [review]

File: doc/release-notes-5262.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+Version 3 of Coinbase will include the following two fields:
+- bestCLHeightDiff (uint32)
+- bestCLSignature (BLSSignature)
+
+bestCLSignature will hold the best Chainlock signature known at the moment of mining.
+bestCLHeightDiff is equal to the diff in heights from the mined block height to the actual Chainlocked block height.
+
+Although miners are forced to include a version 3 Coinbase, the actual real-time best Chainlock signature isn't required as long as blocks contain a bestCLSignature equal to (or newer than) the previous block.
+
+Note: Until the first bestCLSignature is included in a Coinbase once v20 activates, null bestCLSignature and bestCLHeightDiff values are perfectly valid.
```
[the remark this answers is not in the document]
Comment (ogabrielides) [reply doc/release-notes-5262.md:15]: Good point. Will open a PR for that.

**Review by UdjinM6 — APPROVED** [review]
LGTM

**Review by thephez — APPROVED** [review]

**Review by PastaPastaPasta — APPROVED** [review]
utACK for squash merge

**Pull request merged**